user_data_model/Makefile

```diff
@@ -67,22 +67,10 @@ build-pdm:
 	$(PSYNC_HOME)/modules/navigation/navigation.idl \
 	$(PSYNC_HOME)/modules/control/control.idl \
 	idl/user_data_model.idl > pdm/version.txt
+
 # show the results
 	cat pdm/error.log
 
-# install our example data model into the system
```

Comment:
I think documentation might be the better choice so that the user has a better idea of what installing a custom data model means. Also it would be consistent with the other examples.